.github/workflows/linux.yml

```diff
@@ -222,7 +222,15 @@ jobs:
 
   bencher:
     needs: ["build"]
-    # benches must be release (we will do benches for production profile in servo/servo-nightly-builds)
-    if: ${{ inputs.bencher && inputs.profile == 'release' && github.event_name != 'workflow_dispatch' && github.event_name != 'merge_group' }}
-    uses: ./.github/workflows/linux-bencher.yml
+    if: ${{ inputs.bencher && inputs.profile != 'debug' && github.event_name != 'workflow_dispatch' && github.event_name != 'merge_group' }}
+    uses: ./.github/workflows/bencher.yml
+    with:
+      target: 'linux'
+      profile: ${{ inputs.profile }}
+      binary-path: target/${{ inputs.profile }}/servo
+      file-size: true
+      stripped-file-size: ${{ inputs.profile == 'production' }}
+      # We only evaluate speedometer and dromaeo score in release
+      speedometer: ${{ inputs.profile == 'release' }}
+      dromaeo: ${{ inputs.profile == 'release' }}
```

Comment:
Couldn't speedometer and dromaeo be `release` or production?

Reply:
That depends on whether we want to track the speedometer and dromaeo results for all platforms or not, which would also result in additional complications as bencher workflow currently all ran on Ubuntu.

Reply:
I guess we can track the scores for production profile as well, not sure if warranted atm though.